### PR TITLE
Conda receipe for lsdtt-xtensor-python and lsdtopytools

### PR DESCRIPTION
**Pull request by bgailleton**
Conda receipe for (i) lsdtt_xtensor_python, a package relying on xtensor to bind c++ codes to python and (ii) lsdtopytools, a higher level python oackage to control it and cross it with many other python packages.
Receipes generated with grayskull from the `tar.gz` file on pypi

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bgailleton** [comment]
@conda-forge/staged-recipes Ready for review. Thanks!

File: recipes/lsdtt-xtensor-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - xtensor 
+    - xtensor-python 
+    - numpy >=1.14  # [linux]
+    - numpy  # [not linux]
```
Comment (scopatz) [inline recipes/lsdtt-xtensor-python/meta.yaml:26]: please see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (bgailleton) [reply recipes/lsdtt-xtensor-python/meta.yaml]: Thanks @scopatz !
I think it should be sorted. I got odd, inconsistent errors for building for OSx: the conda-forge-ci-setup package was not found

**Comment by bgailleton** [comment]
Thanks a lot @scopatz !

**Pull request merged**